docs/contributing/style-guide.md

```diff
@@ -0,0 +1,420 @@
+
+If you must use industry jargon, such as "white-box monitoring," in a document, be sure to define the term as clearly and concisely as you can.
+
+> White-box monitoring: Monitoring of a system or application based on the metrics it directly exposes, such as logs.
+
+Avoid emojis whenever possible for the same reasons—they can be difficult to understand immediately and don't translate well.
+
+### Mentioning future releases or features
+
+Documentation is meant to describe the product as-is, not as it will be or could be in the future. Netdata documentation generally avoids talking about future features or products, even if we know they are inevitable.
+
+An exception can be made for documenting beta features that are subject to change with further development.
+
+## Language and grammar
+
+Netdata's documentation should be consistent in the way it uses certain words, phrases, and grammar. The following sections will outline the preferred usage for capitalization, point of view, active voice, and more.
+
+### Capitalization
+
+In text, follow the general [English standards](https://owl.purdue.edu/owl/general_writing/mechanics/help_with_capitals.html) for capitalization. In summary:
+
+- Capitalize the first word of every new sentence.
+- Don't use uppercase for emphasis. (Netdata is the BEST!)
+- Capitalize the names of brands, software, products, and companies according to their official guidelines. (Netdata, Docker, Apache, Nginx)
+- Avoid camel case (NetData) or all caps (NETDATA).
+
+#### Capitalization of 'Netdata' and 'netdata'
+
+Whenever you refer to the company Netdata, Inc., or the open-source monitoring agent the company develops, capitalize **Netdata**.
+
+However, if you are referring to a process, user, or group on a Linux system, you should not capitalize, as by default those are typically lowercased. In this case, you should also fence these terms in an inline code block: `` `netdata` ``.
+
+```
+# Not recommended
+The netdata agent, which spawns the netdata process, is actively maintained by netdata, inc.
+
+# Recommended
+The Netdata agent, which spawns the `netdata` process, is actively maintained by Netdata, Inc.
+```
+
+#### Capitalization of document titles and page headings
+
+Document titles and page headings should use sentence case. That means you should only capitalize the first word.
+
+If you need to use the name of a brand, software, product, and company, capitalize it according to their official guidelines.
+
+Also, don't put a period (`.`) or colon (`:`) at the end of a title or header.
+
+**Document titles**:
+
+```
+# Not recommended
+Getting Started Guide
+
+# Recommended
+Getting started guide
+```
+
+**Page headings**:
+
+```
+# Not recommended
+Service Discovery and Auto-Detection:
+
+# Recommended
+Service discovery and auto-detection
+```
+
+**Capitalization of proper nouns**:
+
+```
+# Not recommended
+Install netdata with docker
+
+# Recommended
+Install Netdata with Docker
+```
```

Comment:
The three sections above will look a lot better in a 3x3 table

| Capitalization | Not recommended | Recommended 
| --- | --- | --- 
| Document titles | Getting Started Guide | Getting started guide